firebase.json

```diff
@@ -276,6 +276,7 @@
       { "source": "/go/ios-camera-plugin-thread-model", "destination": "https://docs.google.com/document/d/1Znjp-BduXIb1Ix3JOE1YKyJi1Z0d6rB-2q6Lt7j8xSo/edit", "type": 301 },
       { "source": "/go/ios-platformview-backdrop-filter-blur", "destination": "https://docs.google.com/document/d/1V7Jc_RGaknrBBPPBBKB8lT7f3PKhYr8sin35MSMFAf4/edit?usp=sharing", "type": 301 },
       { "source": "/go/ios-platform-view-focus", "destination": "https://docs.google.com/document/d/1GG8VfYeuVtTQHiMWQXbrSG5HEUorzqU8IGbNySVJ-bY", "type": 301 },
+      { "source": "/go/ios-plugins-swift-migration", "destination": "https://docs.google.com/document/d/1XsaulkJA6_ZSpM7chkQLhQY25sQqhEMiqHMYzw8H85o", "type": 301 },
```

Comment:
this works: 

https://docs.google.com/document/d/1XsaulkJA6_ZSpM7chkQLhQY25sQqhEMiqHMYzw8H85o/edit?resourcekey=0-_cUjF1c0iBvRLKfV-3gK5A

Reply:
the other ones do not have this resource key tho

Reply:
go/drive-resource-keys#urls-with-resource-keys

Reply:
@hellohuanlin Yep, looks like the resource key did it! Once that's been added in the PR link we should be good to land. Thanks @jmagman!